### DISCUSSION TITLE
HELP: Updating Extension for Vite Support: Addressing Reload Issues During Local Development

### DISCUSSION THREAD

**Discussion by DouglasCalora** [quasarframework/quasar#17665] [Quasar App Extensions]
I have my extension https://github.com/bildvitta/asteroid, and I am updating it to support Vite. However, when developing locally and running the command quasar dev, it goes through a few reloads before the server actually starts up.

In my vite project on localhost:
![image](https://github.com/user-attachments/assets/26c6f40b-29e0-428f-b42a-649f2a18730c)

**Comment by DouglasCalora** [comment]
any help?

**Comment by Dtsiantaris** [comment]
If I had to guess I would say it is probably something inside your index.js
If I where you I would remove parts from the index file until it works correctly then start adding them back step by step.
